packages/phishing-controller/src/PhishingController.test.ts

```diff
@@ -373,103 +558,122 @@ describe('PhishingController', () => {
   });
 
   it('should return positive result for unsafe unicode domain from MetaMask config', async () => {
+    sinon.useFakeTimers(2);
+
     nock(PHISHING_CONFIG_BASE_URL)
-      .get(METAMASK_CONFIG_FILE)
+      .get(METAMASK_STALELIST_FILE)
       .reply(200, {
-        blacklist: ['xn--myetherallet-4k5fwn.com'],
+        blocklist: ['xn--myetherallet-4k5fwn.com'],
         fuzzylist: [],
         tolerance: 0,
-        whitelist: [],
+        allowlist: [],
         version: 0,
+        lastUpdated: 0,
       })
-      .get(PHISHFORT_HOTLIST_FILE)
+      .get(METAMASK_HOTLIST_DIFF_FILE)
       .reply(200, []);
 
     const controller = new PhishingController();
-    await controller.updatePhishingLists();
+    await controller.updateStalelist();
     expect(controller.test('myetherẉalletṭ.com')).toMatchObject({
       result: true,
       type: 'blocklist',
       name: 'MetaMask',
     });
   });
 
-  it('should return negative result for unsafe unicode domain if the MetaMask config returns 500', async () => {
+  it('should return a blocklist result for unsafe unicode domain if the MetaMask config returns 500 - as it falls back to default config', async () => {
```

Comment:
Huh. I was wondering - why didn't it fall back to the default config already? But it looks like we'd override the default with a blank config in this scenario :grimacing: That's not good. This is definitely a positive change.